### PR TITLE
move build to GH actions

### PR DESCRIPTION
**Pull request by deathowl**
Move to GH actions as per #266 also as it's a requirement

- [X] Updated the documentation, if you've changed APIs
- [X] Ensured the test suite passes
- [X] Made sure your code lints
- [X] Completed the Contributor License Agreement ("CLA")

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
```
Comment (fornellas) [inline .github/workflows/build.yml:16]: I hope this is not Python 2, lol.

Comment (deathowl) [reply .github/workflows/build.yml:16]: This is v2 of the setup-python action :D

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ![TestSlide](./docs/testslide_logo.png)
 
-[![Build Status](https://travis-ci.com/facebookincubator/TestSlide.svg?branch=master)](https://travis-ci.com/facebookincubator/TestSlide)
+[![Build Status](https://github.com/facebookincubator/TestSlide/workflows/CI/badge.svg)]
```
Comment (fornellas) [inline README.md:3]: Good catch! I completely forgot about it!

**Review by fornellas — APPROVED** [review]
go go go!

**Pull request merged**